src/KDSoapClient/KDSoapValue.cpp

```diff
@@ -367,6 +367,20 @@ QString KDSoapValue::type() const
     return d->m_typeName;
 }
 
+KDSoapValueList KDSoapValue::split() const
+{
+    KDSoapValueList valueList;
+    if (value().toString().trimmed().isEmpty()) return valueList;
```

Comment:
This line seems slow and unnecessary? Well, the trimmed() makes a difference, but I guess skipping empty list items in the for loop below would solve this too (more generically, even)?